__tests__/app.test.js

```diff
@@ -13,7 +13,7 @@ afterAll(() =>{
    return db.end()
 })
 
-describe('1. GET /api/topics', () => {
+describe('3. GET /api/topics', () => {
```

Comment:
Refactors like this can just be pushed straight to main. Alternatively, make them on the old branch you had, push them up, and merge them from a PR that doesn't wait for approval 👍